be/src/exec/pipeline/hashjoin/hash_join_build_operator.cpp

```diff
@@ -64,10 +65,16 @@ Status HashJoinBuildOperator::set_finishing(RuntimeState* state) {
     auto& partial_in_filters = _join_builder->get_runtime_in_filters();
     auto& partial_bloom_filter_build_params = _join_builder->get_runtime_bloom_filter_build_params();
     auto& partial_bloom_filters = _join_builder->get_runtime_bloom_filters();
+
+    auto mem_tracker = state->query_ctx()->mem_tracker();
```

Comment:
Is `set_finished` not counted in the query memory before?

Reply:
counted in instance-level memory tracker